contracts/contracts/GitcoinGrantsResolver.sol

```diff
@@ -113,28 +170,51 @@ contract GitcoinGrantsResolver is SchemaResolver, AccessControl {
      * @param _to The address to which the balance will be transferred.
      * @param _amount The amount to be transferred.
      */
-    function withdraw(
-        address _token, 
-        address _to, 
-        uint256 _amount
-    ) external {
+    function withdraw(address _token, address _to, uint256 _amount) external {
         if (!hasRole(DEFAULT_ADMIN_ROLE, msg.sender)) {
             revert NotAdmin();
         }
         transferAmount(_token, _to, _amount);
     }
 
+    /// ====================================
+    /// ======= Internal / Private =========
+    /// ====================================
+
     /**
      * @notice Transfer an amount of a token to an address
      * @param _token The token to transfer
      * @param _to The address to transfer to
      * @param _amount The amount to transfer
      */
-    function transferAmount(address _token, address _to, uint256 _amount) internal {
+    function transferAmount(
+        address _token,
+        address _to,
+        uint256 _amount
+    ) internal {
         if (_token == NATIVE) {
             _to.safeTransferETH(_amount);
```

Comment:
why are we passing the amount here and not the whole balance?

Reply:
i think to ensure behaviour is consistent. If we use contract balance and the user passes a smaller amount. It will be misleading

Reply:
Also should we use NATIVE or stick to address(0) @nijoe1 @0xKurt ?

Reply:
def NATIVE
address(0) causes always problems.

the transfer is only to withdraw the contracts balance , right? why not withdraw the whole balance? the contract itself shouldn't hold any balance in general